tests/unit/components/settings/settings.spec.js

```diff
@@ -63,4 +54,21 @@ describe('NavBar', () => {
     const btn = wrapper.findAll('.v-btn')
     expect(btn.length).toBe(2)
   })
+
+  test('Connected Edge device version is shown', () => {
+    const localEdgeVersion = require('@/../package.json').version
+
+    store.state.edgeDevice.version = localEdgeVersion
+    store.state.pnp.peerConnectionStatus = PEER_CONNECTED
+    store.state.pnp.remotePeerId = '1234-1234-1234-1234-1234'
+
+    const component = mount(Settings, {
+      localVue,
+      vuetify,
+      store
+    })
+
+    const versionElement = component.get('#version-element')
```

Comment:
@vickywane this test needs to be rewritten. It should mock an edge device status API fetch that returns and edge version. Then compare the edge device version displayed on the page to the version from the status API response.

Reply:
I'd like to object to this here. 

There is already a [test suite](https://github.com/vickywane/ambianic-ui/blob/update/edge-version/tests/unit/remote/edgeAPI.spec.js#L84) in the `edgeApi.spec.js` file that mocks a request to fetch the edge version.
 
Here, the assertion is on displaying the version in the store in the component.